SplitFiles/base.h

```diff
@@ -0,0 +1,70 @@
+#ifndef _DEFINE_BASE_H
+#define _DEFINE_BASE_H
+
+// 1.) Forward declaration: Here is the name of the structure
+// but member-details are omitted.
+struct BreadBinStruct;
+
+// 2.) typedef of the structure
+// Still no details on the members.
+typedef struct BreadBinStruct  BreadBin;
+
+struct BreadTypeStruct {
+    unsigned char     letter;
```

Comment:
and the char, as an 8 bit should probably go last (as a small memory optimisation)

Reply:
Done